### PR TITLE
update shell readme

### PR DESCRIPTION
**Pull request by oakeyc**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-shell/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+A list of common install issues and their resolutions are available at [install troubleshooting](https://github.com/azure/azure-cli-shell/blob/master/docs/install_troubleshooting.md).
+
+
+## Running
+
+To start the application
+
+```bash
+   $ az-shell
```
Comment (troydai) [inline src/command_modules/azure-cli-shell/README.md]: since it is a command module, wasn't is supposed to be `az shell`?

Comment (oakeyc) [inline src/command_modules/azure-cli-shell/README.md]: Yes, just updated it

**Comment by troydai** [comment]
Looks good to me. @derekbekoe please merge when you deem fit.

**Comment by oakeyc** [comment]
removed those sections

**Comment by troydai** [comment]
I think we should re-record the demo video.

File: src/command_modules/azure-cli-shell/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Azure CLI Interactive Shell
+[![Travis](https://travis-ci.org/Azure/azure-cli-shell.svg?branch=master)](https://travis-ci.org/Azure/azure-cli-shell)
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-shell/README.md]: This Travis CI line can also be removed right?

Comment (oakeyc) [inline src/command_modules/azure-cli-shell/README.md]: Yes

**Pull request merged**